recipes/openfisca-france/LICENSE

```diff
@@ -0,0 +1,661 @@
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
Asked upstream in issue ( https://github.com/openfisca/openfisca-france/issues/1791 ) about getting this packaged so we can drop the vendored copy from here